pkg/cli/images_rm.go

```diff
@@ -2,9 +2,9 @@ package cli
 
 import (
 	"fmt"
-	"github.com/acorn-io/acorn/pkg/client"
-
 	cli "github.com/acorn-io/acorn/pkg/cli/builder"
+	"github.com/acorn-io/acorn/pkg/client"
+	client2 "github.com/acorn-io/acorn/pkg/client"
```

Comment:
Can you help me understand the need for a duplicate import here?

Reply:
```
func (a *ImageDelete) Run(cmd *cobra.Command, args []string) error {
	client, err := a.client.CreateDefault()
	if err != nil {
		return err
	}

	for _, image := range args {
		deleted, err := client.ImageDelete(cmd.Context(), image, &client2.ImageDeleteOptions{Force: a.Force})

		if err != nil {
			return fmt.Errorf("deleting %s: %w", image, err)
		}
		if deleted != nil {
			fmt.Println(image)
		} else {
			fmt.Printf("Error: No such image: %s\n", image)
		}
	}

	return nil
}
```
this was named client already I could change the var name too